### DISCUSSION TITLE
Plot VPC by alternative independent variable e.g. TAD

### DISCUSSION THREAD

**Discussion by jfstanding** [nlmixrdevelopment/nlmixr#482] [Q&A]
Is there an option to plot VPCs by an alternative independent variable?  For example, with PsN/NONMEM would use: -idv=TAD to generate simulation files to make a VPC based on time after dose (listed in the dataset) rather than TIME used as the IDV for modelling.  Can this be done with code such as:

vpc(... , idv = "TAD")?

**Comment by mattfidler** [comment]
Currently it can not be done;  I have opened up a feature request here:

https://github.com/nlmixrdevelopment/nlmixr/issues/483
